predicate.go

```diff
@@ -21,6 +31,15 @@ func (ps predicates) Evaluate(root string) ([]string, error) {
 		for _, p := range ps {
 			matched := false
 			if matched, err = p.Match(root, path); err != nil {
+				if pe, ok := err.(PredicateError); ok {
+					// ignore the directory if encountered a different filesystem
+					if pe.errType == ErrorFSType {
+						return filepath.SkipDir
+					} else {
+						fmt.Println(pe.errMessage)
```

Comment:
Can the errors be written to STDERR?

For example, `fmt.Fprintf(os.Stderr, "error: %s\n", pe.errMessage)`

(I realize we're lacking a convenient logger in this project, but for the time being this will at least not clutter standard out with non-hit bytes / information.)